main.c

```diff
@@ -51,6 +51,7 @@
 #include "save_pass.h"
 #include "echo.h"
 #include "as.h"
+#include "service.h"
```

Comment:
This is already included above (at line 41).
Looks good otherwise.

Reply:
Indeed. Removed and force-pushed.